src/Tribe/Aggregator/Settings.php

```diff
@@ -270,12 +270,12 @@ public function default_settings_import( $origin ) {
 	public function get_url_import_range_options( $title = true ) {
 		$options = array(
 			DAY_IN_SECONDS       => array(
-				'title' => __( 'One day', 'the-events-calendar' ),
-				'range' => __( 'a day', 'the-events-calendar' ),
+				'title' => __( '24 hours', 'the-events-calendar' ),
```

Comment:
I wonder if adding context would be useful here (`'title`' and `'range'`) or, if they'll always be identical now, if we should use a var instead of two l10n calls ... will leave to your discretion 🎩